var/spack/repos/builtin/packages/meson/package.py

```diff
@@ -18,6 +18,7 @@ class Meson(PythonPackage):
 
     maintainers = ["eli-schwartz", "michaelkuhn"]
 
+    version("0.64.1", sha256="1d12a4bc1cf3ab18946d12cf0b6452e5254ada1ad50aacc97f87e2cccd7da315")
```

Comment:
Some time today, 1.0.0 is intended to be released.

Reply:
BTW, it's out now.

Reply:
Awesome! Let me add that as well.

Reply:
And, congrats on the 1.0.0 release @eli-schwartz - that's a milestone!

Reply:
https://nibblestew.blogspot.com/2022/12/after-exactly-10-years-meson-100-is-out.html

Fun stuff. :)